lib/Conversion/StandardToHandshake/StandardToHandshake.cpp

```diff
@@ -1509,22 +1511,36 @@ namespace {
 struct HandshakeInsertBufferPass
     : public HandshakeInsertBufferBase<HandshakeInsertBufferPass> {
 
-  DenseMap<Operation *, bool> opVisited;
-  DenseMap<Operation *, bool> opOnStack;
+  void bufferCyclesStrategy() {
+    auto f = getOperation();
+    DenseSet<Operation *> opVisited;
+    DenseSet<Operation *> opOnStack;
```

Comment:
A stack is not a set, so the terminology is a little confusing here...

Reply:
Updated the name to be more agnostic to the storage type, but left the comments about how this set is being used in a "stack-like" way.